src/main/java/io/split/android/client/cache/SplitCache.java

```diff
@@ -65,34 +83,81 @@ public boolean setChangeNumber(long changeNumber) {
 
     @Override
     public long getChangeNumber() {
+        return mChangeNumber;
+    }
+
+    @Override
+    public Split getSplit(String splitName) {
+        Split split =  mInMemorySplits.get(splitName);
```

Comment:
what happens if it is null?

Reply:
If split  is null, the the it tries go load it from disk. If it still is null, that means that the split doesn't exist and the calling function has to handle null value.